### PR TITLE
also build netrc for nonSCL

### PR DESCRIPTION
**Pull request by evgeni**
this is needed for `rest-client` for nonSCL (#3683)

### CODE REVIEW

File: packages/foreman/rubygem-netrc/rubygem-netrc.spec [hunk 1]
```diff
@@ -51,9 +51,11 @@ cp -a .%{gem_dir}/* \
 
 %check
 pushd .%{gem_instdir}
+%if 0%{?scl}
```
Comment (ekohl) [inline packages/foreman/rubygem-netrc/rubygem-netrc.spec]: Can we also skip the pushd and popd when we're outside of the SCL?

File: packages/foreman/rubygem-netrc/rubygem-netrc.spec [hunk 2]
```diff
@@ -71,6 +73,9 @@ popd
 %{gem_instdir}/test
 
 %changelog
+* Mon Apr 29 2019 Evgeni Golov - 0.11.0-4
+- rebuilt
```
Comment (ekohl) [inline packages/foreman/rubygem-netrc/rubygem-netrc.spec]: I'm sure there's a better description here :)

Comment (evgeni) [reply packages/foreman/rubygem-netrc/rubygem-netrc.spec]: Picky.

Comment (ekohl) [reply packages/foreman/rubygem-netrc/rubygem-netrc.spec]: You know me :)

**Pull request merged**